### ISSUE TITLE
pgstatsmon could poll pg_statio_user_tables and pg_statio_user_indexes

### ISSUE THREAD

**Issue by IanWyszynski** [joyent/pgstatsmon#13]
`pg_statio_user_tables` and `pg_statio_user_indexes` are statistics views that pogstgres exposes which contain information about the number of disk blocks read, and shard buffer hits on a per-relation basis. It would be nice to see whether some long-running housekeeping operations (autovacuums), impact these numbers.   

Specifically it would be nice to have time series data for the values of:
* `pg_statio_user_tables.heap_blks_read`
* `pg_statio_user_tables.heap_blks_hit`
* `pg_statio_user_tables.idx_blks_read`
* `pg_statio_user_tables.idx_blks_hit`

by relation name for each of our tables, and:
* `pg_statio_user_indexes.idx_blks_read`
* `pg_statio_user_indexes.idx_blks_hit`

for each of our indexes.

**Comment by IanWyszynski** [comment]
The change seems to pass the test suite pretty consistently, but some tests appear to flaky (failing assertions with and without this change):
```
assert.js:81
  throw new assert.AssertionError({
  ^
AssertionError: one live tuple
    at /root/pgstatsmon/test/basic.tst.js:226:14
    at next (/root/pgstatsmon/node_modules/vasync/lib/vasync.js:843:6)
    at Immediate.immediate._onImmediate (timers.js:448:18)
    at processImmediate [as _immediateCallback] (timers.js:396:17)
```
Is one example where the test suite expects `pg_stat_database.n_tup_live` to be bumped after an INSERT onto the corresponding table. Most of the time this will be true, but it's possible that the postgres statistics collector might not update it in time (the default update interval is 500 ms, which is what the WAIT_PERIOD global is currently set to). 

I've tested the additional queries added in this change manually by generating some graphs with a prometheus server I have deployed there.
